controllers/mcg_bucket.go

```diff
@@ -59,80 +59,19 @@ func (r *ManagedMCGReconciler) watchBucketClass(object client.Object) {
 
 			return
 		}
-		annotations := object.GetAnnotations()
-		if isCacheEnabled, ok := annotations[McgmsCacheEnabled]; ok && isCacheEnabled == "true" {
-			r.reconcileCacheBucketClass(object)
-		}
 	}
 }
 
-func (r *ManagedMCGReconciler) reconcileCacheBucketClass(object client.Object) {
```

Comment:
We don't need this anymore?

Reply:
> automatic cache BucketClass reaction code removes

Can you elaborate on this a bit? We don't need cache buckets now?

Reply:
@rexagod before frond-end was passing the `single` type BucketClass for cached Bucket and we were creating cached BuketClass  along with OBC and single BucketClass had no purpose, but now if `cache` is selected in frond-end, front-end will create cache type BucketClass and backend only create OBC

Reply:
Oh okay, that's sounds much better. Thank you for clarifying this.